iphone/Maps/UI/Help/AboutController/AboutController.swift

```diff
@@ -492,14 +492,13 @@ private extension AboutController {
       return
     }
     // From iOS 14, it is possible to change the default mail app, and mailto should open a default mail app.
-    if MWMMailViewController.canSendMail() {
-      let vc = MWMMailViewController()
-      vc.mailComposeDelegate = self
-      vc.setSubject(subject)
-      vc.setMessageBody(body, isHTML:false)
-      vc.setToRecipients(toRecipients)
-      vc.navigationBar.tintColor = UIColor.whitePrimaryText()
-      self.present(vc, animated: true, completion:nil)
+    let encodedSubject = subject.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed) ?? ""
+    let encodedBody = body.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed) ?? ""
+    let correctEncodedBody = encodedBody.replacingOccurrences(of: "%0A", with: "%0D%0A")
+    
+    if let url = URL(string: "mailto:\(toRecipients)?subject=\(encodedSubject)&body=\(correctEncodedBody)"),
```

Comment:
This code doesn't work for me... I'm using Spark (`to` field):
<img width="300" alt="image" src="https://github.com/organicmaps/organicmaps/assets/79797627/8db4a210-3838-45fd-bc4b-95178a8c836e">


The Problem is in the `toRecipients` - it is an array of Strings. And to work as a subject it should be joined using `;`.
Please take a look at the code used before for opening other apps:
```swift
    func openGmail(subject: String, body: String, recipients: [String]) -> Bool {
      var components = URLComponents(string: "googlegmail://co")!
      components.queryItems = [
        URLQueryItem(name: "to", value: recipients.joined(separator: ";")),
        URLQueryItem(name: "subject", value: subject),
        URLQueryItem(name: "body", value: body),
      ]

      if let url = components.url, UIApplication.shared.canOpenURL(url) {
        UIApplication.shared.open(url)
        return true
      }
      return false
    }
```

So you can create the same method and handle the default app using URLComponents.
Here is the hardcoded one that I've made just as a reference (needs to be refactored):
  ```swift
  func openDefaultApp(subject: String, body: String, recipients: [String]) -> Bool {
      var components = URLComponents(string: "mailto:\(recipients.joined(separator: ";").addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed) ?? "")")!
      components.queryItems = [
        URLQueryItem(name: "subject", value: subject),
        URLQueryItem(name: "body", value: body),
      ]

      if let url = components.url, UIApplication.shared.canOpenURL(url) {
        UIApplication.shared.open(url)
        return true
      }
      return false
    }
```

As a result:
<img width="300" alt="image" src="https://github.com/organicmaps/organicmaps/assets/79797627/0fe68515-41c0-422d-aac6-0ea2289ab71b">